src/validator/validator.ts

```diff
@@ -72,6 +72,8 @@ export const validator = <
     switch (target) {
       case 'json':
         if (!contentType || !jsonRegex.test(contentType)) {
+          const message = 'missing Content-Type header "application/json"'
+          throw new HTTPException(415, { message })
```

Comment:
```suggestion
          throw new HTTPException(415, { message: ERROR_MESSAGE_JSON })
```

In `bodyLimit` the error message is defined as follows. How about doing the same?
Also, it would be better to start with a capital letter.

https://github.com/honojs/hono/blob/02402400381647cc51bc749e611cfac71c191c25/src/middleware/body-limit/index.ts#L10

```ts
const ERROR_MESSAGE_JSON = 'Missing Content-Type header "application/json"'
```

Reply:
I'm not against it if it's consistent but though the rest of the file doesn't follow that approach.

At the very least we'd need one for the form exception as well.

Reply:
I've moved those two messages to their own consts.